src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_module_constants.h

```diff
@@ -53,7 +53,7 @@ const char kSupportedLights[] = "supportedLights";
 // LightControlCapabilities
 const char kName[] = "name";
 const char kDensityAvailable[] = "densityAvailable";
-const char kRGBColorSpaceAvailable[] = "RGBColorSpaceAvailable";
+const char krgbColorAvailable[] = "rgbColorAvailable";
```

Comment:
This constant can be `kRGBColorSpaceAvailable`, shouldn't have the second letter be lower case.

Reply:
agreed

Reply:
@jacobkeeler Fixed at commit fd9f349304e4d47d25d1fed79a0442a599717de8